frontend/src/app/todo/language/language.component.html

```diff
@@ -0,0 +1,11 @@
+<form>
+	<mat-form-field appearance="fill" class="select-container">
+		<mat-label>Selected Language</mat-label>
```

Comment:
It is funny, you implement translation feature and forget to translate the feature itself... replace hardcoded "Select Language" with transloco pipe